internal/services/v1alpha1/schema.go

```diff
@@ -55,18 +59,27 @@ func NewSchemaServer(ds datastore.Datastore, prefixRequired PrefixRequiredOption
 
 func (ss *schemaServiceServer) ReadSchema(ctx context.Context, in *v1alpha1.ReadSchemaRequest) (*v1alpha1.ReadSchemaResponse, error) {
 	var objectDefs []string
+	revisions := map[string]datastore.Revision{}
```

Comment:
this could also use `make(map[string]datastore.Revision, len(in.GetObjectDefinitionNames())`